python-threatexchange/pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
```

Comment:
Hmm? Are you based on a weird version? https://github.com/facebook/ThreatExchange/blob/main/python-threatexchange/pytest.ini